msteams-platform/apps-in-teams-meetings/build-apps-for-anonymous-user.md

```diff
@@ -9,44 +9,64 @@ ms.localizationpriority: medium
 
 # Build apps for anonymous users
 
-You can build bots, messaging extensions, cards and task modules in your app to engage with anonymous meeting participants.
+Anonymous users don't have an Azure AD identity and aren't federated with a tenant. The anonymous participants are like external users but their identity isn't shown in the meeting. An anonymous user can be a presenter or an attendee but can't be an organizer. You can build bots, messaging extensions, and cards and task modules in your app to engage with anonymous meeting participants.
 
-To test your app's experience for anonymous users, select the URL in the meeting invite and join the meeting from a private browser window.
+For anonymous users to interact with the apps in Teams meetings, ensure the following:
+
+1. Update your [app manifest](#app-manifest-update-for-anonymous-users).
+2. Enable the [anonymous user app interaction](#admin-setting-for-anonymous-user-app-interaction) in Teams admin center.
+
+## App manifest update for anonymous users
+
+To allow anonymous users to interact with the tab app, update the `supportsAnonymousGuestUsers` property to `true` in your app manifest. Following is an example of the manifest:
+
+```json
+
+ "meetingExtensionDefinition": {
+  "supportsAnonymousGuestUsers": true
+ }
+
+```
+
+For more information, see [app manifest schema.](~/resources/schema/manifest-schema.md#meetingextensiondefinition)
+
+## Anonymous user authentication flow
+
+Anonymous users can't be authenticated through Azure Active Directory (Azure AD) authentication or `getAuthToken` from the client SDK as they aren't Azure AD accounts. If you need to authenticate anonymous users, your app must identify anonymous users and provide an alternative authentication experience in the meetings. You can determine if a user is anonymous by validating [user's context](#in-meeting-getcontext-from-teams-client-library).
```

Comment:
Could we document how getAuthToken fails for anonymous users? What's the code returned?

Reply:
This is the error: 

useGetAuthToken: useGetAuthToken: Failed with error - User is not authenticated.

Reply:
Clarified with the Tabs team, there is no error code returned other than the above string of error. 

useGetAuthToken: Failed with error - User is not authenticated.